TweetMate/src/main/java/com/ybsystem/tweetmate/usecases/StatusUseCase.java

```diff
@@ -107,10 +107,13 @@ public void onComplete() {
     public static void favorite(TwitterStatus status) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 20 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS3776)

Reply:
Pending